src/resources/postcss/components/full/_view-selector.pcss

```diff
@@ -181,11 +177,33 @@
 		box-shadow: var(--box-shadow-tooltip);
 	}
 
-	.tribe-events-c-view-selector__list-item--active {
+	.tribe-events-c-view-selector__list-item {
 
-		.tribe-events-c-view-selector__list-item-link {
-			background-color: var(--color-background-secondary);
+		.tribe-common--breakpoint-medium& {
+
+			&.tribe-events-c-view-selector__list-item--active {
+				display: none;
+			}
+
+			&:first-child:nth-last-child(1),
+			&:first-child:nth-last-child(2),
+			&:first-child:nth-last-child(2) ~ li,
+			&:first-child:nth-last-child(3),
+			&:first-child:nth-last-child(3) ~ li {
+				&.tribe-events-c-view-selector__list-item--active {
+					display: block;
+				}
+			}
```

Comment:
This is all skeleton stuff. But also, what is this `first-child` and `nth-last-child` stuff supposed to be doing?

Reply:
Skeleton - you are correct.
Selection based on if there are 3 or less - since the number of elements is dynamic (1-7).